core/src/main/java/com/linecorp/armeria/client/ClientFactory.java

```diff
@@ -177,6 +178,11 @@ static void disableShutdownHook() {
      */
     MeterRegistry meterRegistry();
 
+    /**
+     * Returns the {@link PrometheusMeterRegistry} that collects various DNS stats.
+     */
+    PrometheusMeterRegistry dnsMetricRegistry();
+
```

Comment:
I don't think we need this at all. We can use the same `MeterRegistry` for DNS metrics.

Reply:
I have removed it.